defaults/aws_config.yaml

```diff
@@ -9,23 +9,23 @@ instance_type_monitor: 't3.large'
 # manual on creating loader AMI's: see docs/new_loader_ami.md
 regions_data:
   us-east-1: # US East (N. Virginia)
-    ami_id_loader: 'ami-00c1a16b5136fbb7c' # Loader dedicated AMI v19
-    ami_id_monitor: 'ami-02eac2c0129f6376b' # Official CentOS Linux 7 x86_64 HVM EBS ENA 1901_01
+    ami_id_loader: 'ami-00c1a16b5136fbb7c' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-06878d265978313ca' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
   us-west-2: # US West (Oregon)
-    ami_id_loader: 'ami-04d2eb44d523b3ccb' # Loader dedicated AMI v19
-    ami_id_monitor: 'ami-01ed306a12b7d1c96' # Official CentOS Linux 7 x86_64 HVM EBS ENA 1901_01
-  eu-west-1: # Europe (Ireland)
-    ami_id_loader: 'ami-042cf1bc21e30ce60' # Loader dedicated AMI v19
-    ami_id_monitor: 'ami-0ff760d16d9497662' # Official CentOS Linux 7 x86_64 HVM EBS ENA 1901_01
+    ami_id_loader: 'ami-09808043b7fcdb244' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-03f8756d29f0b5f21' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  eu-west-1: # EU (Ireland)
```

Comment:
Here `Europe` is changed to `EU`, but lines `20` and `23` stay with `Europe`.
So, need either to revert this change back or update those.
Personally, I don't see a need to change it.

Reply:
It's a change in comment, cause of the code I've taken from siren-tests.
nitpicking from my POV.

Reply:
It is totally normal to reduce the number of changes in a huge PR.